corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -208,6 +208,12 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
 
         templateHelpers: function () {
             var appId = Util.currentUrlToObject().appId;
+            var tempData = this.options.model.get('data');
+            for (var i =0; i < tempData.length; i++) {
+                if(this.options.styles[i] === "graph") {
+                    tempData[i] = tempData[i][1];
+                }
+            }
```

Comment:
this is pretty confusing, where is tempData used and why do we need to do this for just the graph style?